integration/agent/api/src/main/java/com/splunk/rum/integration/agent/api/SplunkRum.kt

```diff
@@ -101,11 +107,13 @@ class SplunkRum private constructor(
     }
 
     companion object {
-        private val noop = SplunkRum(
+        @Deprecated("Instance is available if install is not called prior to fetching instance.")
+        val noop = SplunkRum(
```

Comment:
Why is this public? Since we are discontinuing the API, it should be enough to have:

```kotlin
@JvmStatic
@Deprecated("Use SplunkRum.instance without calling SplunkRum.install() to get noop instance")
fun noop(): SplunkRum {
    return noop
}
```